css/style.css

```diff
@@ -1,81 +1,263 @@
-    font-family: "Poppins", sans-serif;
-  }
-
-  header {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px;
-  }
-
-  h1 {
-    margin: 0;
-  }
-
-  #menu-icon {
-    display: block;
-    position: absolute;
-    right: 20px;
-    top: 10px;
-    cursor: pointer;
-    padding-top: 10px;
-  }
-
-  nav a {
-    text-decoration: none;
-    color: #6070ff;
-    font-size: 18px;
-    line-height: 20px;
-    font-weight: 700;
-  }
-
-  #headline {
-    background: url("../images/header-shapes\ mobile.png");
-    padding: 20px;
-  }
-
-  #headline h1 {
-    color: #172b4d;
-    font-size: 40px;
-    font-weight: 700;
-    line-height: 52px;
-    margin-top: 92px;
-  }
-
-  #headline p {
-    color: #344563;
-    font-size: 16px;
-    font-weight: 400;
-    line-height: 24px;
-  }
-
-  #headline span {
-    text-decoration: none;
-    color: #7f8cff;
-    font-size: 16px;
-    font-weight: 500;
-    line-height: 24px;
-    text-transform: uppercase;
-  }
-
-  #social-links img {
-    width: 20px;
-    height: 20px;
-    padding: 10px 15px 1px 1px;
-    cursor: pointer;
-  }
-}
-
-/* Desktop screen */
-@media (min-width: 768px) {
-  body {
-    background-color: aliceblue;
-  }
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700&display=swap");
+
+html {
+  margin: 0;
+  padding: 0;
+  background: white;
+  font-family: "Poppins", sans-serif;
+}
+
+button {
+  font-weight: 500;
+  line-height: 24px;
+  color: #396df2;
+  border: solid #6070ff 1px;
+  border-radius: 8px;
+  padding: 12px;
+  width: 126px;
+  height: 48px;
+  margin: 10px 1px;
+  background: white;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: white;
+  box-shadow: 0.1px 8px 16px 0.1px #4053fc3d;
+}
+
+button:active {
+  background: #2230d2;
+  color: white;
+}
+
+button:disabled {
+  color: #c1c7d0;
+  background: #c1c7d0;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+
+h1 {
+  margin: 0;
+}
+
+#menu-icon {
+  display: block;
+  position: absolute;
+  right: 20px;
+  top: 10px;
+  cursor: pointer;
+  padding-top: 10px;
+}
+
+nav a {
+  text-decoration: none;
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 700;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  background: url("../images/header-shapes\ mobile.png");
+}
+
+/* Project section */
+
+#project-container {
+  display: grid;
+  grid-template-columns: auto;
+  background: #f7f7f9;
+  padding: 20px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  max-width: 327px;
+  margin: 15px;
+  padding: 16px;
+  margin-top: 88px;
+  border-radius: 16px;
+  background-color: #fff;
+  border: solid #dfe1e6 1px;
+  cursor: pointer;
+}
```

Comment:
- [ ] [OPTIONAL] I notice that your card is not properly aligned  🗃️ and responsive 📱 . However, the previous reviewer didn't mention it so I'm assigning it as an OPTIONAL change. Though I **HIGHLY** recommend you to implement it because it will improve the design of your webpage 📄

## Your design in a screen 608px 👇🏽 Note the spacing at the right side 😁 

![Screenshot from 2023-07-08 10-28-43](https://github.com/shimels1/portfolio_microverse/assets/10439283/e3af77fc-5ebe-4720-aa2b-5ed0a6d88739)

You might fix it using this snippet of code:
```
.card {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  margin: 88px auto 15px auto;
  padding: 16px;
  border-radius: 16px;
  background-color: #fff;
  border: solid #dfe1e6 1px;
  cursor: pointer;
}
```